### PR TITLE
Page Local Navigation: remove (deprecated)

### LINKED ISSUE #3205: [Housekeeping] Remove Page Local Navigation
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the housekeeping request
Local Navigation was marked as experimental, but out of caution we have chosen to first deprecate it in v9. 
For v10 page local navigation should be removed entirely as it is replaced by tab navigation.

### PR DESCRIPTION
**Pull request by jakobe**
## Which issue does this PR close?

This PR closes #3205

## What is the new behavior?

Removed the deprecated `PageLocalNavigation` component and the (now empty) experimental examples.

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Reminders
- [x] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [x] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [ ] Determine if your changes are a fix, feature or breaking-change, and add the matching label to your PR. If it is tooling, dependency updates or similar, add ignore-for-release.
- [x] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [ ] Request that the changes are code-reviewed 
- [ ] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be merged to `develop` by Team Kirby.

### CODE REVIEW

**Review by RasmusKjeldgaard — CHANGES_REQUESTED** [review]
Awesome! I agree we should remove the experimental module as well 👍

File: MIGRATION.md [hunk 1]
```diff
@@ -179,7 +179,6 @@ Additionally, multiple components and directives are now standalone components,
 - ComponentLoaderDirective
 - ThemeColorDirective
 - SlideButtonComponent
-- PageLocalNavigationComponent
```
Comment (RasmusKjeldgaard) [inline MIGRATION.md]: This should probably not be removed from the migration-notes for v8 😊

**Review by RasmusKjeldgaard — COMMENTED** [review]
One nitty gritty thing!

File: MIGRATION.md (hunk 1 above)
Comment (jakobe) [reply MIGRATION.md]: Reverted 👍🏻

**Review by RasmusKjeldgaard — APPROVED** [review]

**Pull request merged**